browser.go

```diff
@@ -121,7 +123,8 @@ func CreateWindow(_, _ int, title string, monitor *Monitor, share *Window) (*Win
 			go w.framebufferSizeCallback(w, w.canvas.Width, w.canvas.Height)
 		}
 		if w.sizeCallback != nil {
-			go w.sizeCallback(w, int(w.canvas.GetBoundingClientRect().Width), int(w.canvas.GetBoundingClientRect().Height))
+			boundingW, boundingH := w.GetSize()
```

Comment:
I don't know it very well - but isn't it a risk to move from the canvas bounding rect to the window size?

Reply:
I don't know, but I can actually just use the Width and Height that has been calculated in this listener it seems.